src/GPToggle.tsx

```diff
@@ -16,61 +18,39 @@ type GPToggleProps = {
 };
 
 export default function GPToggle(props: GPToggleProps) {
-  const handleGP = (event: React.MouseEvent<HTMLElement>, nextGP: string) => {
-    if (nextGP) {
-      props.setGP(nextGP);
-      props.setScoreHeight("level1");
-      props.setReefTarget(-1);
-    }
+  const handleGP = (
+    event: React.ChangeEvent<HTMLInputElement>,
+    isCoral: boolean
+  ) => {
+    let nextGP = isCoral ? "coral" : "algae";
+    props.setGP(nextGP);
+    props.setScoreHeight(nextGP == "algae" ? "level2" : "level4");
```

Comment:
Algae should also default to L4 (net) imo. In smart autonomy we don't even pay attention to L2/3 for algae since the heights are pre-determined